src/website/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width">
     <meta name="description" content="Mineral UI is a design system and React component library for the web that lets you quickly build high-quality, accessible apps. Created by CA Technologies.">
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i&subset=cyrillic,greek" rel="stylesheet">
+    <link href="/index.css" rel="stylesheet">
```

Comment:
This is not necessary: https://github.com/mineral-ui/mineral-ui/blob/7d6f2ab63196f8819fe745b853bc8d3c2135157a/src/website/index.js#L26

Reply:
Whoa! That inline preview is super nice.

Reply:
addressed

Reply:
Wow, that is cool.